releases/UWP/net-native2.0/README.md

```diff
@@ -4,8 +4,16 @@ You can see what was included in each .NET Native 2.0 ([Microsoft.NETCore.Univer
 
 When using Visual Studio these packages require Visual Studio 2017 Version 15.4 or later.
 
-### 6.0.1 (October 5, 2017)
+### UWP 6.0.2 (.NET Native 2.0.1) (November 20, 2017)
+
+#### Quality fixes
+- Fixes an issue when building UWP apps that target RS3 and referenced the Microsoft Store Services SDK and  Microsoft Advertising SDK or Store Engagement SDK [514498]
```

Comment:
referenced --> reference

Reply:
double space between SDK and<blank><blank>Microsoft Advertising...